docs/user-guide/authenticating-with-client-certificates.md

```diff
@@ -71,6 +71,17 @@ Register the client certificate with the user ID in your ESM. The following comm
 This metadata can be used for TLS client authentication.
 :::
 
+## Enable the internal API ML mapper
```

Comment:
I don't think it belongs here, I believe the section should just note that the configuration information moved to the relevant Configuration article.

Reply:
Ok. Thanks for taking a look. I'll just link to it in the client cert article.